guides/common/modules/proc_creating-custom-snippets-to-apply-errata.adoc

```diff
@@ -0,0 +1,36 @@
+[id="creating-custom-snippets-to-apply-errata_{context}"]
+= Creating custom snippets to apply errata
+
+Custom snippets allow you to execute custom code when applying errata on hosts.
+You can run code before and/or after applying errata.
+
+.Prerequisites
+* Depending on the used template, multiple custom snippet hooks exist which you can use to include custom snippets.
+Ensure that you check your template first to verify which custom snippets you can use.
++
+You can view used job templates under *Administer* > *Remote Execution Features*.
+
+.Procedure
+. In the {ProjectWebUI}, navigate to *Hosts* > *Templates* > *Job Templates*.
+. Click *Create Template*.
+. In the *Name* field, enter a name for your custom snippet.
+The name has to start with the name of a template that supports including custom snippets:
+** Append ` custom pre` to the name of a template to run code before applying errata on hosts.
+** Append ` custom post` to the name of a template to run code after applying errata on hosts.
+. On the *Type* tab, select *Snippet*.
+. Click *Submit* to create your custom snippet.
+
+[id="cli-creating-custom-snippets-to-apply-errata_{context}"]
+.CLI procedure
+. Before you create a template with the CLI, create a plain text file that contains your custom snippet.
```

Comment:
```suggestion
. Create a plain text file that contains your custom snippet.
```
I don't think you need the first clause because it is a numbered step.

Reply:
Makes sense; applied.